packages/reports/addon/components/navi-date-picker.js

```diff
@@ -174,9 +174,9 @@ export default Ember.Component.extend({
    * @returns {boolean} true if date is the same
    */
   _isNewDateValue(newDate) {
-    let lastTime = get(this, '_lastTimeDate');
+    let lastTime = get(this, 'lastTimeDate');
```

Comment:
`lastTimeDate`? should this be called `previousDate`?

Reply:
There's a previousDate already being used by `didReceiveAttrs`. I didn't write this component, but after playing with it, it seems that both properties are necessary.